datahub/search/company/test/test_entity_search_views.py

```diff
@@ -47,37 +51,59 @@ def setup_data(es_with_collector):
     country_us = constants.Country.united_states.value.id
     country_anguilla = constants.Country.anguilla.value.id
     uk_region = constants.UKRegion.south_east.value.id
-    CompanyFactory(
+    company1 = CompanyFactory(
         name='abc defg ltd',
         trading_names=['helm', 'nop'],
         address_1='1 Fake Lane',
         address_town='Downtown',
         address_country_id=country_uk,
         registered_address_country_id=country_uk,
         uk_region_id=uk_region,
-        export_to_countries=[
-            constants.Country.france.value.id,
-        ],
-        future_interest_countries=[
-            constants.Country.japan.value.id,
-            constants.Country.united_states.value.id,
-        ],
     )
-    CompanyFactory(
+    CompanyExportCountryFactory(
+        company=company1,
+        country=Country.objects.get(pk=constants.Country.france.value.id),
```

Comment:
Can do something like this instead:

```suggestion
        country_id=constants.Country.france.value.id,
```

(same for the other ones below).